CODEOWNERS

```diff
@@ -0,0 +1,43 @@
+# Live DX Team
+
+.github/              @ledgerhq/live-devx
+scripts/              @ledgerhq/live-devx
+tools/                @ledgerhq/live-devx
+fastlane/             @ledgerhq/live-devx
+
+# Live Blockchain Support Team
+
+apps/**/families/                                               @ledgerhq/live-blockchain-support
+ledger-live-desktop/src/live-common-set-supported-currencies.*  @ledgerhq/live-blockchain-support
+ledger-live-mobile/src/live-common-setup.*                      @ledgerhq/live-blockchain-support
+ledger-live-common/src/families                                 @ledgerhq/live-blockchain-support
+ledgerjs/packages/cryptoassets                                  @ledgerhq/live-blockchain-support
+ledgerjs/packages/types-cryptoassets                            @ledgerhq/live-blockchain-support
+ledgerjs/packages/hw-app-*                                      @ledgerhq/live-blockchain-support
+
+# Live Platform and Services Team
+
+ledger-live-desktop/src/renderer/screens/exchange/  @ledgerhq/live-platform-and-services
+ledger-live-mobile/src/screens/Exchange/            @ledgerhq/live-platform-and-services
+ledger-live-common/src/exchange/                    @ledgerhq/live-platform-and-services
+ledger-live-common/src/platform/                    @ledgerhq/live-platform-and-services
```

Comment:
Should we separate the platform team and the services team in github as we're now different teams ?
Or is it only for the platform team and we just need to rename the team ?

Reply:
Valid point. Whenever those teams are created we can update